op-challenger/metrics/metrics.go

```diff
@@ -84,3 +107,27 @@ func (m *Metrics) RecordUp() {
 func (m *Metrics) Document() []opmetrics.DocumentedMetric {
 	return m.factory.Document()
 }
+
+func (m *Metrics) ClearTrackedGames() {
+	m.trackedGames.Delete(prometheus.Labels{
+		"status": GameStatusInProgress,
+	})
+	m.trackedGames.Delete(prometheus.Labels{
+		"status": GameStatusChallengerWon,
+	})
+	m.trackedGames.Delete(prometheus.Labels{
+		"status": GameStatusDefenderWon,
+	})
```

Comment:
We don't want to delete - we want to set to 0.  The game statuses still exist, we just have no games in that status.